serializer/custom_normalizer.rst

```diff
@@ -12,21 +12,20 @@ Creating a New Normalizer
 Imagine you want add, modify, or remove some properties during the serialization
 process. For that you'll have to create your own normalizer. But it's usually
 preferable to let Symfony normalize the object, then hook into the normalization
-to customize the normalized data. To do that, leverage the ``ObjectNormalizer``::
+to customize the normalized data.::
 
     // src/Serializer/TopicNormalizer.php
     namespace App\Serializer;
 
     use App\Entity\Topic;
     use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
     use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
-    use Symfony\Component\Serializer\Normalizer\ObjectNormalizer;
 
     class TopicNormalizer implements NormalizerInterface
     {
         public function __construct(
             private UrlGeneratorInterface $router,
-            private ObjectNormalizer $normalizer,
+            private NormalizerInterface $normalizer,
```

Comment:
I think this leads to a cyclic dependency, if I remember correctly when I documented something similar :thinking: You may base yourself on those ones I think!

- https://github.com/symfony/symfony-docs/pull/18602
- https://github.com/symfony/symfony-docs/pull/18603

Reply:
Indeed, I missed that part. I think we just need to backport https://github.com/symfony/symfony-docs/pull/18603 to 6.3 and 6.4 branches 🙂

Reply:
Indeed, if forgot about that part, I copied the #18602 PR contents, thanks!
Btw, this is deprecated since 6.1 (and not 6.4), should we update the 6.4 branch as well?

Reply:
I'd say yes, I guess this _may_ be managed during upmerge?

Reply:
Yes we will handle it while unmerging this PR 👍